examples/Images and Tables.ipynb

```diff
@@ -219,7 +231,36 @@
    },
    "outputs": [],
    "source": [
-    "tv"
+    "from firefly_widgets import ChartViewer"
```

Comment:
Very nice to have the ChartViewer! I suggest to add a Markdown cell explaining that the table in 'tv' is the source of data for the chart, and that filtering the table will update the chart. It took me some minutes to figure this out myself.